src/Map.elm

```diff
@@ -226,19 +243,23 @@ listToArray mapList =
     List.foldl mapper (Array.fromList []) mapList
 
 
-coordinateOnMap : ( Int, Int ) -> ( Int, Int )
-coordinateOnMap ( lat, lon ) =
-    ( mapHeight - lat, mapHeight - lon )
-
+coordinateOnMap : ( Int, Int ) -> Int -> ( Int, Int )
+coordinateOnMap ( lat, lon ) mapSize =
+    let
+        toHalf =
+            mapSize // 2 - 1
+    in
+    ( toHalf - lat, toHalf - lon )
 
-getLandscape : ( Int, Int ) -> Maybe LandscapeId
-getLandscape position =
-    getItemFrom2dArray (coordinateOnMap position) worldMap
 
+getLandscape : ( Int, Int ) -> World Height -> Maybe LandscapeId
```

Comment:
These `( Int, Int )` pairs you have would be a good opportunity for a `Point` or `Coord` type.

Reply:
Thank you, will check!